tool_collections/samtools/samtools_bedcov/samtools_bedcov.xml

```diff
@@ -1,8 +1,9 @@
-<tool id="samtools_bedcov" name="Calculate read depth" version="1.0.0">
-    <description>on BAM files</description>
-    <requirements>
-        <requirement type="package" version="1.1">samtools</requirement>
-    </requirements>
+<tool id="samtools_bedcov" name="Calculate read depth" version="1.0.1">
+  <description>on BAM files</description>
+  <macros>
+    <import>macros.xml</import>
+  </macros>
+  <expand macro="requirements"></expand>
     <version_command>samtools --version | head -n 1 | awk '{ print $2 }'</version_command>
     <command><![CDATA[
```

Comment:
I think you can also move the version_command into the macro. `sam_to_bam` is missing the version_command for example.